project/core/templates/landing.html

```diff
@@ -26,6 +26,7 @@
 </head>
 
 <body>
+{%block content%}
```

Comment:
Since this doesn't extend from `base.html`, we don't need to define the content block.
```suggestion
```